frame/ibc/primitives/src/lib.rs

```diff
@@ -59,19 +60,22 @@ pub struct SendPacketData {
 	PartialOrd,
 	scale_info::TypeInfo,
 )]
-pub struct OffchainPacketType {
+pub struct PacketInfo {
+	pub height: Option<u64>,
 	pub sequence: u64,
 	pub source_port: Vec<u8>,
 	pub source_channel: Vec<u8>,
 	pub destination_port: Vec<u8>,
 	pub destination_channel: Vec<u8>,
+	pub channel_order: u8,
```

Comment:
why Is `channel_order` sometimes an `u8`, sometimes a `String`?

Reply:
oh yeah the substrate runtime doesn't support strings